src/github.com/getlantern/lantern-android/client/conn.go

```diff
@@ -0,0 +1,63 @@
+package client
+
+import (
+	"errors"
+	"net"
+)
+
+var (
+	// ErrCouldNotCreateListener is returned when NewListener fails.
+	ErrCouldNotCreateListener = errors.New(`Could not create new listener.`)
+	// ErrClosed is returned when the server is closed.
+	ErrClosed = errors.New(`Server was manually closed.`)
+)
+
+// Listener is a wrapper around net.TCPListener that attempts to provide a
+// Stop() function.
+type Listener struct {
```

Comment:
As a rule of thumb, I prefer to not export anything unless it needs to be used outside of its package.  Listener seems like it's completely internal to the client package.